doc/source/serve/pipeline.rst

```diff
@@ -0,0 +1,99 @@
+.. _serve-pipeline-api:
+
+Pipeline API (Experimental)
+===========================
+
+This section should help you:
+
+- understand the experimental pipeline API
+- build on top of the API to construct your multi-model inference pipelines.
+
+
+.. note::
+    This API is experimental and the API is subject to change.
+    We are actively looking for feedback via the Ray Slack or Github issues.
+
+Basic API
+---------
+
+Serve Pipeline is a standalone package that can be used without Ray Serve. 
+However, the expected usage is to use it inside your Serve deployment.
```

Comment:
kind of a weird thing to open with. I'd start with the problem statement: This package is mean to make it seamless to develop and deploy multi-model pipelines. Add a sentence or two about what the advantages are (all in code, good perf)